### PR TITLE
Prepare catalog script to be moved to boilerplate

### PR DESCRIPTION
**Pull request by rporres**
I want the script to be copied in boilerplate with minimal changes, so here's some preparation

* We use ensure to install opm and build
* We use repo_root mechanism used in boilerplate
* Some editing in variable naming, to make them clearer

Signed-off-by: Rafa Porres Molina <rporresm@redhat.com>

### CODE REVIEW

File: hack/build_channel_catalog.sh [hunk 1]
```diff
@@ -45,49 +41,17 @@ done
...
+$repo_root/boilerplate/openshift/golang-osd-operator/ensure.sh grpcurl
+grpcurl_local_executable="$repo_root/.grpcurl/bin/grpcurl"
 
 # ./hack/generate-operator-bundle-contents.py
-HERE=${0%/*}
-if [[ ! -x "$HERE/generate-operator-bundle-contents.py" ]]; then
-    echo "$HERE/generate-operator-bundle-contents.py is either missing or non-executable"
+bundle_contents_cmd="$repo_root/hack/generate-operator-bundle-contents.py"
+if [[ ! -x "$bundle_contents_cmd" ]]; then
+    echo "$bundle_contents_cmd is either missing or non-executable"
```
Comment (2uasimojo) [inline hack/build_channel_catalog.sh:54]:
```suggestion
    log "$bundle_contents_cmd is either missing or non-executable"
```
?

**Review by 2uasimojo — APPROVED** [review]
This is sane. I didn't go and check all the variable renames by hand.

/lgtm

File: hack/build_channel_catalog.sh (hunk 1 above)
Comment (rporres) [reply hack/build_channel_catalog.sh:54]: For sure, I don't know what I was thinking 🤦

**Pull request merged**